go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	github.com/celestiaorg/rsmt2d v0.6.0
 	github.com/regen-network/cosmos-proto v0.3.1
 	github.com/tendermint/tendermint v0.34.20
+	golang.org/x/exp v0.0.0-20221012211006-4de253d81b95
```

Comment:
this was added to import the type constraint https://pkg.go.dev/golang.org/x/exp@v0.0.0-20221012211006-4de253d81b95/constraints#Integer